Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -1102,6 +1115,19 @@ namespace AzToolsFramework
         }
     }
 
+    void PropertyRowWidget::CreateGroupToggleSwitch()
+    {
+        if (!m_toggleSwitch)
+        {
+            m_handlerName = AZ::Edit::UIHandlers::CheckBox;
+            EBUS_EVENT_RESULT(m_handler, PropertyTypeRegistrationMessages::Bus, ResolvePropertyHandler, m_handlerName, azrtti_typeid<bool>());
+            m_toggleSwitch = m_handler->CreateGUI(this);
+            m_middleLayout->insertWidget(0, m_toggleSwitch, 1);
+            auto checkBoxCtrl = reinterpret_cast<AzToolsFramework::PropertyCheckBoxCtrl*>(m_toggleSwitch);
```

Comment:
suggestion: I think this can just be a `static_cast` (or we should use a Qt dynamic_cast to ensure it's the right type)

Reply:
Sounds good, static_cast works well!